core/domain/topic_services.py

```diff
@@ -834,6 +835,9 @@ def delete_topic(committer_id, topic_id, force_deletion=False):
         committer_id, feconf.COMMIT_MESSAGE_TOPIC_DELETED,
         force_deletion=force_deletion)
 
+    feedback_services.delete_threads_for_multiple_entities(
+        feconf.ENTITY_TYPE_TOPIC, [topic_id])
```

Comment:
Do we create any thread for topic?

Reply:
There is the possibility to create a suggestion for new question in a topic and all suggestions are created with threads.

Reply:
We create suggestions for the skill entity!